public/components/integrations/components/create_integration_helpers.ts

```diff
@@ -204,7 +206,8 @@ export const doExistingDataSourceValidation = async (
 
 const createComponentMapping = async (
   componentName: string,
-  payload: ComponentMappingPayload
+  payload: ComponentMappingPayload,
```

Comment:
Can you please create an issue in `https://github.com/opensearch-project/opensearch-js/issues` to add missing components in the js client.